public/app/features/dashboard/state/PanelModel.ts

```diff
@@ -41,6 +42,8 @@ const notPersistedProperties: { [str: string]: boolean } = {
   cachedPluginOptions: true,
   plugin: true,
   queryRunner: true,
+  replaceVariables: true,
+  getDatasourceConfig: true,
```

Comment:
do think this needs to be here, only replaceVariables that is bound on the instance